### PR TITLE
fail2ban-client: add page

### PR DESCRIPTION
**Pull request by antoniobusrod**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Since I have only used `fail2ban-client` command in Linux systems, I have not placed in common. I have seen that `fail2ban-client` can be installed in Mac with [MacPorts](https://ports.macports.org/port/fail2ban/summary), but since Mac installation is not supported in [Wiki installation documentation page](https://github.com/fail2ban/fail2ban/wiki/How-to-install-or-upgrade-fail2ban-manually), I have just included it in `linux` directory.

Moreover, I have documented the main options I usually use, for example, to manage `nginx-http-auth` jail.

### References
* [fail2ban commands](https://www.fail2ban.org/wiki/index.php/Commands)
* [GitHub repository](https://github.com/fail2ban/fail2ban/)
* [fail2ban-client(1) - Linux man page](https://linux.die.net/man/1/fail2ban-client)

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]
LGTM! Thanks @antoniobusrod.

File: pages/linux/fail2ban-client.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# fail2ban-client
+
+> Configure and control fail2ban server.
+> More information: <https://github.com/fail2ban/fail2ban/>.
+
+- Retrieve current status of the jail service:
+
+`fail2ban-client status {{jail}}`
+
+- Manually unban IP from the jail service:
```
Comment (navarroaxel) [inline pages/linux/fail2ban-client.md:10]: I think the `manually` is not needed:
```suggestion
- Unban IP from the jail service:
```
I'm sure @sbrl won't like the `unban` as a verb, maybe he knows a good synonym.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/fail2ban-client.md (hunk 1 above)
Comment (owenvoke) [reply pages/linux/fail2ban-client.md:10]: I feel like this is probably fine (to use _unban_) as it's the terminology used by Fail2ban. 🤷🏻

Comment (sbrl) [reply pages/linux/fail2ban-client.md:10]: Unban is fine as a verb here, I think. However, I think we're missing an article. What  about this?
```suggestion
- Unban an IP from the jail service:
```

However, I like @owenvoke's description much better, so I think we should go with that instead.
```suggestion
- Remove the specified IP from the jail service's ban list:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**